### ISSUE TITLE
Require S3 for SCS-compatible KaaS 

### ISSUE THREAD

**Issue by mbuechse** [SovereignCloudStack/standards#727]
We recently noticed that KaaS relies on S3 (and maybe other APIs, such as LBaaS?)

Lay this down in a standard.

**Comment by mbuechse** [comment]
@jschoone I'm assigning this to you mainly to check whether I got this right. We still have to designate someone to implement this (in alignment with Team Container, of course).

**Comment by mbuechse** [comment]
Okay, I don't know for sure, but the LB could already be handled in https://github.com/SovereignCloudStack/issues/issues/227

**Comment by jschoone** [comment]
> @jschoone I'm assigning this to you mainly to check whether I got this right. We still have to designate someone to implement this (in alignment with Team Container, of course).

Yes it's fine, I'll take a close look and classify such topics after R7.
